torchvision/transforms/transforms.py

```diff
@@ -2120,15 +2120,15 @@ def get_params(alpha: List[float], sigma: List[float], size: List[int]) -> Tenso
         dy = dy * alpha[1] / size[1]
         return torch.concat([dx, dy], 1).permute([0, 2, 3, 1])  # 1 x H x W x 2
 
-    def forward(self, tensor: Tensor) -> Tensor:
```

Comment:
Could you add back the typing info here?

Reply:
There is no docstring in this case. We can just revert the code change on this file.

Reply:
If I revert the code, the parameter is named `tensor` of type tensor.
But the existing docstring says "img (PIL Image or Tensor): Image to be transformed" - wrong parameter name and type.

Reply:
The reason why I recommend reverting is because the input tensor can also be a video with the timestamp on the `dim=-4`. This is why it's preferred to keep the original code and completely remove the change on this file.